### PR TITLE
Feat/dashboard link to extension docs

### PR DESCRIPTION
**Pull request by GeekyADAMS**
## What kind of change does this PR introduce?
Supabase Studio > Database > Extensions
feature: enables linking database extensions from the dashboard to their docs

## What is the current behavior?
No external link to database extension docs from the dashboard
[https://github.com/supabase/supabase/issues/15415](https://github.com/supabase/supabase/issues/15415)

## What is the new behavior?
Can now link to database extension doc from the dashboard:

Screenshot 2023-06-28 at 21 16 26
Screenshot 2023-06-28 at 21 16 32

### CODE REVIEW

File: studio/components/interfaces/Database/Extensions/ExtensionCard.tsx [hunk 1]
```diff
@@ -69,15 +70,29 @@ const ExtensionCard: FC<Props> = ({ extension }) => {
...
+            <a className="max-w-[85%] cursor-default" target="_blank" rel="noreferrer">
+              <div className="flex flex-row items-center">
+                <h3
+                  title={extension.name}
+                  className="h-5 m-0 text-base uppercase truncate cursor-pointer text-scale-1200"
+                >
+                  {extension.name}
+                </h3>
+
+                <IconExternalLink className="ml-2.5 cursor-pointer" size={18} />
```
Comment (saltcod) [inline studio/components/interfaces/Database/Extensions/ExtensionCard.tsx:91]:
```suggestion
                <IconExternalLink className="ml-2.5 cursor-pointer" size={14} />
```

**Comment by saltcod** [comment]
@GeekyADAMS sadly won't be as simple as this. Some of the extensions don't yet have docs.  We prob need to add a new constant to store something like name and docsUrl so we can use it there. I don't _think_ one exists now.

**Comment by saltcod** [comment]
Also related
https://github.com/supabase/supabase/pull/15253

**Comment by gregnr** [comment]
Worth mentioning we have https://github.com/supabase/supabase/blob/master/apps/docs/data/extensions.json, but it's only under docs currently.

**Comment by GeekyADAMS** [comment]
Saw the extensions.json file @gregnr, but it looked incomplete and I thought to use the data used to list extensions on the dashboard already (from this source: [https://api.supabase.io/platform/pg-meta/crjjxiuvwrfygigomyuo/extensions](https://api.supabase.io/platform/pg-meta/crjjxiuvwrfygigomyuo/extensions))

**Comment by saltcod** [comment]
We should prob move that to packages then @gregnr ?

**Comment by gregnr** [comment]
Yes that would be best. Let's wait for #15253 to merge, then we can move that in this PR.

**Review by kiwicopple — APPROVED** [review]

**Pull request merged**